cmd/prompt_test.go

```diff
@@ -0,0 +1,72 @@
+//go:build linux
+// +build linux
+
+package cmd
+
+import (
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hinshun/vt10x"
+
+	"knative.dev/kn-plugin-func/docker"
+)
+
+func Test_newPromptForCredentials(t *testing.T) {
+	expectedCreds := docker.Credentials{
+		Username: "testuser",
+		Password: "testpwd",
+	}
+
+	console, _, err := vt10x.NewVT10XConsole()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer console.Close()
+
+	go func() {
+		_, _ = console.ExpectEOF()
+	}()
+
+	go func() {
+		chars := expectedCreds.Username + enter + expectedCreds.Password + enter
```

Comment:
@matejvasek sorry for my ignorance here, but where does `enter` come from? I only see it twice in this file and it appears to be uninitalized, but I know that can't be the case or it would not compile.